### PR TITLE
Fix Symfony 5.3 deprecations

### PR DESCRIPTION
**Pull request by keksa**
closes https://github.com/1up-lab/OneupUploaderBundle/issues/407

### CODE REVIEW

**Comment by keksa** [comment]
@bytehead failing tests look unrelated

**Comment by bytehead** [comment]
But somehow sf5 related 🙃

**Comment by keksa** [comment]
They fail exactly the same way on `master` too. The only thing I found is that the `annotation_reader` service should be defined by the FrameworkBundle, but that's already there, so no idea. I guess I'll leave that up to you.

**Comment by bytehead** [comment]
It is somehow related to Symfony 5.3, it works on 5.2

**Comment by bytehead** [comment]
@keksa can you rebase on master?

**Comment by keksa** [comment]
@bytehead done

File: src/Controller/AbstractController.php [hunk 1]
```diff
@@ -66,7 +66,7 @@ public function progress(): JsonResponse
         $request = $this->getRequest();
 
         /** @var SessionInterface $session */
```
Comment (bytehead) [inline src/Controller/AbstractController.php:68]: Are those typehints still needed?

Comment (keksa) [reply src/Controller/AbstractController.php]: missed the unused namespaces, should be good to go now

**Comment by bytehead** [comment]
Thank you @keksa 🙏

**Comment by bytehead** [comment]
https://github.com/1up-lab/OneupUploaderBundle/releases/tag/3.1.3

**Pull request merged**